normandy/recipes/tests/test_filters.py

```diff
@@ -175,6 +178,72 @@ def test_throws_error_on_bad_platform(self):
             filter.to_jexl()
 
 
+class TestPrefCompareFilter(FilterTestsBase):
+    def create_basic_filter(
+        self, pref="browser.urlbar.maxRichResults", value=10, comparison="equal"
+    ):
+        return PrefCompareFilter.create(pref=pref, value=value, comparison=comparison)
+
+    def test_generates_jexl(self):
+        filter = self.create_basic_filter()
+        assert filter.to_jexl() == "10 == 'browser.urlbar.maxRichResults'|preferenceValue"
+
+    def test_generates_jexl_greater_than(self):
+        filter = self.create_basic_filter(comparison="greater_than")
+        assert filter.to_jexl() == "10 < 'browser.urlbar.maxRichResults'|preferenceValue"
+
+    def test_generates_jexl_greater_than_equal(self):
+        filter = self.create_basic_filter(comparison="greater_than_equal")
+        assert filter.to_jexl() == "10 <= 'browser.urlbar.maxRichResults'|preferenceValue"
+
+    def test_generates_jexl_less_than(self):
+        filter = self.create_basic_filter(comparison="less_than")
+        assert filter.to_jexl() == "10 > 'browser.urlbar.maxRichResults'|preferenceValue"
+
+    def test_generates_jexl_less_than_equal(self):
+        filter = self.create_basic_filter(comparison="less_than_equal")
+        assert filter.to_jexl() == "10 >= 'browser.urlbar.maxRichResults'|preferenceValue"
```

Comment:
These tests are fine. I'd like to show you a trick we could use to make them more concise. I'll leave it up to you which version you'd prefer though.


```suggestion
    @pytest.mark.parametrize("comparison,symbol", [
      ("greater_than", ">"),
      ("greater_than_equal", ">="),
      ("less_than", "<"),
      ("less_than_equal", "<="),
    ])
    def test_generates_jexl_number_ops(self, comparison, symbol):
        filter = self.create_basic_filter(comparison=comparison)
        assert filter.to_jexl() == f"10 {symbol} 'browser.urlbar.maxRichResults'|preferenceValue"
```

Reply:
😮 TIL: this trick as well as suggested change!

Reply:
I made this change. I decided to not use Github's automatic "commit suggestion" button though, so I could change the comparison direction back.